### ISSUE TITLE
Simulated keyboard input with multiple Windower instances

### ISSUE THREAD

**Issue by Petewilmer** [Windower/Issues#888]
Since the latest hook went live, there is an issue with simulating keyboard input using the windower console commands on windower instances that do not have focus.

`send_command('setkey numpad7 down;wait 0.1;setkey numpad7 up;')`

The above code snippet will trigger and disable follow when used (via lua) on my main, but if I use the same command on alts that do not have focus, the command does not work. This used to work, so I'm sure the issue is with the new hook rather than how the command is being sent to the windower console. i.e. I'm not doing:
`send @others setkey numpad7 down;wait 0.1;setkey numpad7 up;`
I'm doing:
`send @others gs c stop follow`
and in the GS file of the other characters, it's triggering:
`send_command('setkey numpad7 down;wait 0.1;setkey numpad7 up;')`
I have put in a 'print('test')' to confirm that the code is being reached still (it is), and I have manually triggered gs c stop follow by alt tabbing directly to an alt and doing //gs c stop follow. When manually triggered on the alt, with the window in focus, it works.

**Comment by z16** [comment]
We have a new option inside the Windower launcher (start *Windower* -> *Select profile* -> *Edit* -> *Game*) to enable sound and gamepad controls when the window isn't active (*Always Play Sound* and *Always Enable Gamepad*). Try enabling gamepad controls and see if it helps. I know it's not a gamepad you're having issues with, but it might be related to how SE handles cross-instance input.

**Comment by Petewilmer** [comment]
Just tried it with gamepad always enabled but sadly it's still not working on instances out of focus :(

**Comment by Petewilmer** [comment]
What's interesting is if I make a .txt script that does:
`setkey numpad7 down;wait 0.1;setkey numpad7 up;`
And then use `send_command('exec stop-follow.txt;')` then it works perfectly. The issue appears to be with send_command() not doing `setkey numpad7 down;wait 0.1;setkey numpad7 up;` whilst out of focus. So for now I can do what I need using .txt console scripts.

**Comment by Petewilmer** [comment]
Been playing around with this more this morning. It seems that the setkey command only works for my alts when the console window is open. If I leave the console window open on all alts then the `setkey numpad7 down;wait 0.1;setkey numpad7 up;` works fine. As soon as I close the console, it stops working again. I've had Byrth suggest to me that this is because of alt+tab remaining pressed down maybe, so I tried adding the following to the stop-follow.txt:
`setkey lalt up;setkey alt up;setkey tab up;`
But that didn't work sadly. I also tried no ever pressing alt+tab and instead clicking between windows, but that didn't work either (also I normally play borderless fullscreen so that would be quite a change to get used to).
Let me know if there are any other tests I can try to run.
